apps/finance/contracts/Finance.sol

```diff
@@ -616,11 +599,15 @@ contract Finance is EtherTokenConstant, IsContract, AragonApp {
         uint256 _amount,
         uint256 _paymentId,
         uint64 _paymentRepeatNumber,
-        string _reference
+        string _reference,
+        bool _skipChecks
     )
         internal
     {
-        require(_getRemainingBudget(_token) >= _amount, ERROR_REMAINING_BUDGET);
+        if (!_skipChecks) {
+            require(_getRemainingBudget(_token) >= _amount, ERROR_REMAINING_BUDGET);
+        }
```

Comment:
Is there a reason to not write it as:
```
require(_skipChecks || _getRemainingBudget(_token) >= _amount, ERROR_REMAINING_BUDGET);
```
?

Reply:
I would actually move this check to `newPayment` so we can avoid the use of this `_skipChecks` parameter at all.